tutorbackup/templates/backup/build/backup/restore_local_services.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+tar -xf backup/backup.tar.gz
```

Comment:
Probably want to include `-v` here as well, and also consider the gzip vs. xz comment on the other script.

Reply:
Will do